### PR TITLE
Issue 95   add possibility to order parties

### PR DESCRIPTION
**Pull request by Riscum**
option to order parties alphabetical added. 
Used switch component and painted checked versions o black instead of blue.

### CODE REVIEW

**Comment by Riscum** [comment]
page load
no order

ordered
ordered

un-order again
no oreder again

File: app/javascript/components/home/HomePartiesList.js [hunk 1]
```diff
@@ -15,17 +15,42 @@ limitations under the License.
...
     }
 
     render() {
-        const { parties } = this.props;
+        let { parties } = this.props;
+        let { ordered } = this.state;
+
+        if (ordered) {
 
+            parties.sort(function (a, b) {
```
Comment (luisiul359) [inline app/javascript/components/home/HomePartiesList.js]: You have a function in `utils` that can help you here:

```
import { sortArrayByKey } from '../../utils';

parties = sortArrayByKey(parties.map(x => {x["keyToSort"] = x.title.toUpperCase(); return x}), "keyToSort");
```

**Review by luisiul359 — COMMENTED** [review]
Please merge with master and solve the conflicts.

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
+// Place all the styles related to the home controller here.
```
Comment (inesrosete) [inline app/assets/stylesheets/home.scss]: Merge issue, this file can be removed.

**Review by inesrosete — APPROVED** [review]

**Pull request merged**